programs/connected/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "connected"
```

Comment:
Couldn't be named example instead?

Reply:
Or maybe include the test keyword in the program name?